content/docs/atom-guide/dev-guide/shaders/azslc/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+linktitle: AZSLc
+title: AZSLc, The Amazon Shading Language Compiler
+description: Learn about the Amazon Shading Language Compiler (AZSLc) for Atom Renderer. 
+weight: 100
+---
+
+**Amazon Shading Language Compiler (AZSLc)** is the front end compiler of the Atom Renderer shader build pipeline. AZSLc is open source, and supports Windows, macOS, and Linux. The source code for AZSLc is available on GitHub here: [Amazon Shading Language Compiler](https://github.com/o3de/o3de-azslc).
+
+AZSLc serves the following two purposes:
+* Transpile shaders written in **Amazon Shading Language (AZSL)** into high-level shader language (HLSL).
+* Extract, in the form of JSON files, reflection data from the `.azsl` files including the shader constants layout, resource binding information, shader variant options, and more.
+
+New and updated `.azsl` source assets that are placed in project or Gem scan directories are automatically detected by **Asset Processor**. Shader asset builders provide process job information for the `.azsl` source assets to Asset Processor. When `.azsl` source assets are processed, AZSLc transpiles the shader source to high-level shader language (HLSL). The shader product assets are compiled for the specified target platforms by DirectX Shader Compiler (DXC) and SPIRV-Cross.
```

Comment:
```suggestion
AZSLc, platform-specific shader compilers, shader asset builders, and **Asset Processor** work together to process `.azsl` source assets into product assets that the engine can consume. Asset Processor automatically detects new and updated `.azsl` source assets that are placed in project or Gem scan directories. Shader asset builders provide process job information about the `.azsl` source assets to Asset Processor. When `.azsl` source assets are processed, AZSLc transpiles the shader source to HLSL. The shader product assets are compiled for the specified target platforms by DirectX Shader Compiler (DXC) and SPIRV-Cross.
```
* Consider adding the first sentence to give a "big picture" of the process that you're explaining here. In my suggestion, I list the process being achieved and the different tools that are involved. I think a sentence like this is effective for learning because it simplifies the complex process that is the shader asset pipeline.

Reply:
Fixed.